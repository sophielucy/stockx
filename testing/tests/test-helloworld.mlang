int main()
{
   print(1);
   printb(true);
   printf(1.0);
   prints("Hello World!");
   return 0;
}
