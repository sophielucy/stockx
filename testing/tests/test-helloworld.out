1
1
1.000000
"Hello World!"
